Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -83,6 +84,69 @@ constexpr float attachmentIconSize = 72;
 constexpr float attachmentIconSize = 52;
 #endif
 
+#if !RELEASE_LOG_DISABLED
+static CString compactStackTrace(StackTrace& stackTrace)
```

Comment:
can you add a comment / documentation on what the contents of the return value actually are / what format the compact stack trace is in?

Reply:
also, why Cstring and not normal String?

Reply:
`stackTrace.forEachFrame` gives us 8-bit `const char*` C strings (which I truncate to just the function name), so I assumed `CString` would be the most appropriate type. Do you think `String` would be better?